execution/executor-types/src/transactions_with_output.rs

```diff
@@ -64,31 +67,31 @@ impl TransactionsWithOutput {
     }
 
     pub fn get_last_checkpoint_index(&self) -> Option<usize> {
+        if self.is_reconfig {
+            return Some(self.len() - 1);
+        }
+
         (0..self.len())
             .rev()
-            .find(|&i| Self::need_checkpoint(&self.transactions[i], self.epoch_ending_flags[i]))
+            .find(|&i| self.transactions[i].is_non_reconfig_block_ending())
```

Comment:
why this is not the last one? for chunks?

Reply:
yes, for state sync